src/test/groovy/org/prebid/server/functional/tests/pricefloors/PriceFloorsCurrencySpec.groovy

```diff
@@ -397,7 +397,7 @@ class PriceFloorsCurrencySpec extends PriceFloorsBaseSpec {
 
      def "PBS should choose floorMin from imp[0].ext.prebid.floors when imp[0].ext.prebid.floors is present"() {
         given: "Default BidRequest with floorMin, floorMinCur"
-        def impExtPrebidFloorMin = PBSUtils.randomFloorValue
+        def impExtPrebidFloorMin = PBSUtils.getRandomFloorValue(FLOOR_MAX, FLOOR_MIN + FLOOR_MAX)
```

Comment:
Why not `getRandomFloorValue(FLOOR_MIN,FLOOR_MAX)`?

Reply:
Need be confident that `impExtPrebidFloorMin` will be biggest than another `PBSUtils.randomFloorValue`